src/newsView.test.js

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
need to add 
 /**
 * @jest-environment jsdom
 */
for it to run jest properly for test